### ISSUE TITLE
Reintroduce a way to join nodes from the console

### ISSUE THREAD

**Issue by marianoguerra** [riak-core-lite/rebar3_template_riak_core_lite#1]
currently in the makefile there are commands that used the console and admin support in riak core to join, we need to change it to reintroduce support for it.

@albsch how do you do it? using rpc commands from the shell?

**Comment by albsch** [comment]
The way we use it in AntidoteDB is to hide it from the user and provide API functions to create a DC (for example with a [Go script wrapped in an easy to use docker container](https://github.com/AntidoteDB/antidote-connect/blob/master/antidote-connect.go). We left the decision on how to join clusters to the application which uses `riak_core`, i.e. AntidoteDB uses `riak_core` calls internally. 

If you want to skip defining an API call, a simple way to replace the `riak-admin` commands in the Makefile would be as follows:

- `$(APPNAME)-admin cluster join node1@127.0.0.1` -> `$(APPNAME) eval 'riak_core:join("node1@127.0.0.1")'`
- `$(APPNAME)-admin cluster plan` -> `$(APPNAME) eval 'riak_core_claimant:plan()'`
- `$(APPNAME)-admin cluster commit` -> `$(APPNAME) eval 'riak_core_claimant:commit()'`

For `member-status`, I think we'll have to reintroduce the nice formatting print solution, which was cut out when I removed `clique`.

**Comment by marianoguerra** [comment]
@albsch I just pushed a change that makes `make devrel` work again, I can start a cluster and join the nodes.

The only problem is that the ring doesn't seem to rebalance, do you have an idea why? I will look into it tomorrow.

regarding member-status, we can move it into the riak_core_lite_utils module we mentioned and include it in the template, what do you think?

**Comment by albsch** [comment]
I'll take a look at that, too.

**Comment by albsch** [comment]
So, one thing is that `riak_core:join(...)` triggers an auto join. I thought I made staged joins mandatory, need to look into that.

But for my case, with both `join` and `staged_join`+`plan`+`commit` and checking with `riak_core_status:ring_ready()` I see the ring rebalancing. Leaving and rejoining and so on works with the ring getting rebalanced, too (I tried with your current template). Any more details on your case?
